src/main/java/org/eyeseetea/sdk/presentation/fileio/FileIOUtils.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2017.
+ *
+ * This file is part of QA App.
+ *
+ *  Health Network QIS App is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Health Network QIS App is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eyeseetea.sdk.presentation.fileio;
+
+
+import android.content.Context;
+import android.content.res.AssetFileDescriptor;
+import android.graphics.Bitmap;
+import android.media.MediaMetadataRetriever;
+import android.net.Uri;
+import android.util.Log;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.channels.FileChannel;
+
+public class FileIOUtils {
```

Comment:
How we have talked other times, utils class is not a good idea because is a mix of concepts, is true that some times has a sense as on this, but in this class, we have mixing file utilities, video utilities, Database utilities. Can you refactor this class to split it among FileUtils, VideoUtils, and DatabaseUtils?